### DISCUSSION TITLE
No access to github copilot after verification

### DISCUSSION THREAD

**Discussion by Setzubun** [community/community#75436] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body
EDIT: It is working now, I did not know that I had to enable it.

It says here that my request has gotten through. However, I don't have access to it in visual studio code. I have logged out and closed vs code but it still isn't working.

<img width="242" alt="image" src="https://github.com/community/community/assets/147253433/f0119e53-f949-4f3d-92b6-dc28b498022e">
<img width="940" alt="image" src="https://github.com/community/community/assets/147253433/df6c06d8-a7f8-4f7c-bec0-d6abaaa4168e">
<img width="344" alt="image" src="https://github.com/community/community/assets/147253433/1728f08c-070d-4d68-99ec-3909275d6203">

**Comment by sy9870** [comment]
Hi! I'm in the same situation as you. How were you able to enable it? I've already authorized Copilot but I'm still getting the same errors. Thanks

**Reply by davecheney** [reply, marked as answer]
@sy9870 you do not appear to have an active copilot subscription. You can sign up for copilot here https://github.com/features/copilot/
